### DISCUSSION TITLE
Does this Aim Bot Application work for Blood Strike?

### DISCUSSION THREAD

**Discussion by RishBroProMax** [Babyhamsta/Aimmy#1466] [General]
Does this Aim Bot Application work for Blood Strike?

**Comment by MozerLins** [comment]
Sim , acabei de jogar perfeitamente

**Comment by lostaccesstomygithub** [comment]
do NOT use this on bloodstrike, you will get banned.

**Reply by RishBroProMax** [reply]
> do NOT use this on bloodstrike, you will get banned.

why brooo
